tests/modules/images/regenerate-existing-images/helper-tests.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Tests for regenerate-existing-images module helper.php.
+ *
+ * @package performance-lab
+ * @group regenerate-existing-images
+ */
+
+class Regenerate_Existing_Images_Helper_Test extends WP_UnitTestCase {
```

Comment:
When i run this class unit test using `wp-env run phpunit 'phpunit -c /var/www/html/wp-content/plugins/performance/phpunit.xml.dist --verbose' --filter Regenerate_Existing_Images_Helper_Test` it's show error like below.

`Error: Class 'Perflab_Background_Job' not found`

Reply:
Because WP unit tests are not really tests in isolation. You can use group `regenerate-existing-images` to only test this module.

Reply:
```
wp-env run phpunit 'phpunit -c /var/www/html/wp-content/plugins/performance/phpunit.xml.dist --verbose --group=regenerate-existing-images'
```